nestkernel/event.h

```diff
@@ -1062,6 +1075,450 @@ class GapJunctionEvent : public SecondaryEvent
   double get_coeffvalue( std::vector< unsigned int >::iterator& pos );
 };
 
+/**
+ * Event for rate model connections without delay.
+ */
+class RateNeuronEvent : public SecondaryEvent
```

Comment:
How much of the code here could be moved to `SecondaryEvent`? There seems quite a bit of overlap with other of these classes.